test/test-network/systemd-networkd-tests.py

```diff
@@ -5252,7 +5252,7 @@ def test_dhcp_client_ipv6_only(self):
         self.assertIn('DHCPADVERTISE(veth-peer)', output)
         self.assertIn('DHCPREQUEST(veth-peer)', output)
         self.assertIn('DHCPREPLY(veth-peer)', output)
-        self.assertNotIn('rapid-commit', output)
+        self.assertNotIn('14:rapid-commit', output)
```

Comment:
Please add short comment about this is for DHCPv6, and DHCPv4 also has rapid commit option with another option value.

Reply:
I just removed this change. It was only necessary in the first place b/c I accidentally had rapid-commit in the request options, which is incorrect and illegal anyway.